apps/meeting/src/app.tsx

```diff
@@ -57,6 +42,44 @@ const App: FC = () => (
   </Router>
 );
 
+const MeetingView: React.FC = () => {
+  const { addVoiceFocus } = useVoiceFocus();
+  const { meetingMode } = useAppState();
+  const [isVoiceFocusChecked, setIsVoiceFocusChecked] = useState(false);
+
+  const reselection = (device: Device): Promise<Device | VoiceFocusTransformDevice> => {
+    if (isVoiceFocusChecked) {
+      return addVoiceFocus(device);
+    }
+    return Promise.resolve(device);
+  };
```

Comment:
Why is this not the same kind of code that's in the input control, using the meeting manager to find out if VF is enabled, and preserving that across reselection?

There is definitely going to be duplicate code, so I wonder if we can break it out as a context?